lib/screens/home/tabs/contact_us/contact_us.dart

```diff
@@ -27,23 +33,312 @@ class ContactUsScreen extends StatelessWidget {
+                                wordSpacing: 4),
+                          ),
+                        )
+                      ],
+                    ),
+                  ),
+                  Container(
+                    padding: EdgeInsets.only(top: 50 * heightFactor, right: D.horizontalPadding),
+                    child: Row(
+                      children: [
+                        Expanded(
+                          flex: 1,
+                          child: GestureDetector(
+                            onTap: () => _launchURL(S.linkedinURL),
+                            child: Image.asset(
+                              S.assetLinkedinIcon,
+                              width: 60,
+                              height: 60,
+                            ),
+                          ),
+                        ),
+                        Expanded(
+                          flex: 1,
+                          child: GestureDetector(
+                            onTap: () => _launchURL(S.twitterURL),
+                            child: Image.asset(
+                              S.assetTwitterIcon,
+                              width: 60,
+                              height: 60,
+                            ),
+                          ),
+                        ),
+                        Expanded(
+                          flex: 1,
+                          child: GestureDetector(
+                            onTap: () => _launchURL(S.facebookURL),
+                            child: Image.asset(
+                              S.assetFacebookIcon,
+                              width: 60,
+                              height: 60,
+                            ),
+                          ),
+                        ),
+                        Expanded(
+                          flex: 1,
+                          child: GestureDetector(
+                            onTap: () => _launchURL(S.instagramURL),
+                            child: Image.asset(
+                              S.assetInstagramIcon,
+                              width: 60,
+                              height: 60,
+                            ),
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                  Container(
+                    alignment: Alignment.centerLeft,
+                    padding: EdgeInsets.only(top: 50 * heightFactor),
+                    child: RichText(
+                      text: TextSpan(
+                        text: "Feedback",
+                        style: TextStyle(fontSize: 45 * heightFactor, fontWeight: FontWeight.w900),
+                      ),
+                    ),
+                  ),
+                  Container(
+                    padding: EdgeInsets.only(top: 20 * heightFactor),
+                    child: Row(
+                      children: [
+                        RichText(
```

Comment:
this can be just Text. Use rich text when you have a line(s) with different parts having different font properties.